### PR TITLE
new: success flashing on completion of char can be have its color cha…

### LINKED ISSUE #49: Final flashing color distinct from stroke hint flashing color
The final flashing color should definitively be separated from the hint as visually they represent 2 conflictual visual communication:
- a stroke hint appears when you were wrong after X times... it is a correction. Common colors that could come to mind is redish colors.
- the final victory flash is a confirmation that you are right. People might want to use green or yellow bright colors. But not refer to the colors used when you actually failed.

Does this make sense ? I can work on that in a few days if this seems okay for you.

### PR DESCRIPTION
**Pull request by vaab**
…nged.

This implementation is probably a little naïve, I hope it'll help somehow. It fixes #49

### CODE REVIEW

File: src/HanziWriter.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ const defaultOptions = {
   highlightColor: '#AAF',
   outlineColor: '#DDD',
   drawingColor: '#333',
+  highlightCompleteColor: '#0ff',
```
Comment (chanind) [inline src/HanziWriter.js]: Could this be `null` by default? That way it won't change behavior for existing users

File: src/HanziWriter.js [hunk 2]
```diff
@@ -98,6 +99,7 @@ HanziWriter.prototype.setOptions = function(options) {
   });
   this._highlightCharOptions = assign({}, this._mainCharOptions, {
     strokeColor: this._options.highlightColor,
+    highlightCompleteColor: this._options.highlightCompleteColor,
```
Comment (chanind) [inline src/HanziWriter.js]: It doesn't look like this is used by the `CharacterRenderer` class - no need for this line

File: src/HanziWriter.js (hunk 1 above)
Comment (vaab) [reply src/HanziWriter.js]: Yes, of course.

File: src/HanziWriter.js (hunk 2 above)
Comment (vaab) [reply src/HanziWriter.js]: Oh, good catch. Sorry for that.

**Comment by chanind** [comment]
Closing as this is implemented in https://github.com/chanind/hanzi-writer/pull/74

**Pull request closed without merging**